cdap-cli/src/main/java/co/cask/cdap/cli/CLIMain.java

```diff
@@ -92,22 +151,40 @@ public boolean handleException(PrintStream output, Exception e, int timesRetried
           output.println("Error: " + e.getMessage());
         }
 
+        if (verbose) {
+          e.printStackTrace(output);
```

Comment:
This doesn't look right. A user shouldn't see stack traces right? *verbose* should be for seeing verbose output messages. But stack traces is a more 'debug' thing?